with-vue-supabase/pages/index.js

```diff
@@ -0,0 +1,69 @@
+import Head from 'next/head'
+import Image from 'next/image'
+import styles from '../styles/Home.module.css'
+
+export default function Home() {
```

Comment:
It doesn't seem like you are fetching data from the StepZen GraphQL API.. Could you implement some data fetching? Now it looks like just an API and an app that exist in the same directory, there's no communication between the two..